src/text/Text.js

```diff
@@ -44,8 +44,8 @@ function Text(text, style)
 
     core.Sprite.call(this, core.Texture.fromCanvas(this.canvas));
 
-    this.setText(text);
-    this.setStyle(style);
+    this.text = text;
+    this.style = style;
```

Comment:
Even though these will technically set the underscored tracker variables, we should put them in the ctor explicitly as well and document them as private members.